File: BeerViewer/Models/kcsapi/kcsapi_charge.cs
namespace BeerViewer.Models.kcsapi
{
	public class kcsapi_charge
	{
		public kcsapi_charge_ship[] api_ship { get; set; }
		public int[] api_material { get; set; }
		public int api_use_bou { get; set; }
	}
}

File: WindowFrame/src/KanColle/Raw/kcsapi_charge.ts
// tslint:disable:class-name
export interface kcsapi_charge {
	api_ship: kcsapi_charge_ship[];
	api_material: number[];
	api_use_bou: number;
}
export interface kcsapi_charge_ship {
	api_id: number;
	api_fuel: number;
	api_bull: number;
	api_onslot: number[];
}
